sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/DefaultWizard.java

```diff
@@ -555,8 +555,10 @@ private boolean isJava8orAbove(String javaVersion) {
 		if (javaVersion.startsWith("1.8")) {
 			return true;
 		}
-		int pos = javaVersion.indexOf('.');
-		String version = javaVersion.substring(0, pos);
```

Comment:
Instead of doing contains() and then indexOf, isn't more efficient to just do substring basing on the return value of indexOf?

Reply:
I updated the PR.